eyeshade/workers/reports.js

```diff
@@ -109,6 +118,48 @@ const daily = async (debug, runtime) => {
   debug('daily', 'running again ' + moment(tomorrow).fromNow())
 }
 
+exports.freezeOldSurveyors = freezeOldSurveyors
+
+async function freezeOldSurveyors (olderThanDays, anchorTime, surveyors) {
+  // in seconds
+  const nDays = DAY_SECONDS * olderThanDays
```

Comment:
you might consider extracting the time comparison logic into a function that takes `olderThanDays`, `anchorTime` and a mongo id. prime target for some unit testing if you do so